builds/android/app/src/main/java/org/easyrpg/player/player/EasyRpgPlayerActivity.java

```diff
@@ -83,6 +88,7 @@ protected void onCreate(Bundle savedInstanceState) {
 				getWindow().setFlags(0x01000000, 0x01000000);
 			}
 		} catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Okay, I never see them because Intellij convert them silently.
The java convention is 4 spaces (I hate this, but well), do you want me to convert this file with spaces or tabulation?
http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html